src/views/AddItem.jsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import { isEmpty, isDuplicate } from '../utils/validateStrings';
 import { addItem } from '../api/firebase';
+import NoToken from '../components/NoToken';
```

Comment:
Love that this is also its own component so that the conditional rendering doesn't get too convoluted.